components/Home/Services/ServicesCard/ServicesCard.tsx

```diff
@@ -0,0 +1,39 @@
+import ComposedTitles from "@components/Common/ComposedTitles";
+import PrimaryDescription from "@components/Common/PrimaryDescription";
+import ServicesSvg from "@components/Common/ServicesSvg";
+import { ServicesInfo } from "@lib/types";
+
+interface ServicesCardProps {
+  data: ServicesInfo;
+  index: number;
+}
+const ServicesCard = ({ data, index }: ServicesCardProps) => {
+  const { id, title, description } = data;
+  const printSvgTop = () => {
+    if (id == 1) {
+      return <ServicesSvg />;
+    }
+    return "";
+  };
+  const printSvgBottom = () => {
+    if (id == 3) {
+      return <ServicesSvg bottom />;
+    }
+    return "";
+  };
+  return (
+    <div className="relative w-full px-4 mb-12 lg:w-1/2">
+      {printSvgTop()}
+      {printSvgBottom()}
+      <span className="flex items-center justify-center w-12 h-12 mb-6 text-sm rounded-full lg:mb-8 bg-blue-50">
+        {index + 1}
```

Comment:
@oscarcorellan Let's move this operation: `{index + 1}` to a constant variable: 

`const currentIndex = index + 1;`

Generally we will try to avoid doing JS operations like this inside of the JSX, and instead save them in variables and only use variables inside the JSX

```suggestion
        {currentIndex}
```